server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -419,66 +407,6 @@ public Path matchingFileSystem(Path source, String[] options) {
     return null;
   }
 
-  @Override
-  public Path getFullPath(TableId tableId, String path) {
```

Comment:
There is another method in this class named `getVolumeByPath` that has relative path handling code.  Would that be something to address in this PR?  Or should that be another PR?

Reply:
I think ```getVolumeByPath``` may have to be done in another change, related to the DEFAULT volume code.  That method returns the DEFAULT volume if it detects a relative path, which I could remove but I wanted to look more into whether the DEFAULT volume can be removed with the other legacy properties.